src/opnsense/mvc/tests/app/models/OPNsense/Base/Constraints/ComparedToFieldConstraintTest.php

```diff
@@ -78,6 +78,55 @@ public function test_if_it_validates_number_ranges_correctly_with_gt_and_error()
         $this->assertEquals(true, $ret);
     }
 
+    // Empty values
```

Comment:
ah so I guess these are the tests I'm missing? :)

Reply:
They are the tests from the other PR, they will be removed from this one after rebase.